.github/workflows/soroban-rpc.yml

```diff
@@ -111,7 +111,7 @@ jobs:
     env:
       SOROBAN_RPC_INTEGRATION_TESTS_ENABLED: true
       SOROBAN_RPC_INTEGRATION_TESTS_CAPTIVE_CORE_BIN: /usr/bin/stellar-core
-      PROTOCOL_20_CORE_DEBIAN_PKG_VERSION: 20.2.0-1716.rc3.34d82fc00.focal
+      CORE_DEBIAN_PKG_VERSION: 20.4.1-1807.b152dc51d.focal
```

Comment:
Do we need to have some sort of directive (maybe in the Captive Core config) to tell Core to run with Protocol 21 enabled? Or is this already implied by this build (which is weird given the `20` in the version..)

Reply:
It's implicit for captive core when it connects to the network but we also have the protocol number hardcoded in the simulation library (we should probably automate that)

Reply:
https://github.com/stellar/soroban-rpc/issues/137